sparse/client.go

```diff
@@ -127,7 +130,7 @@ func (client *syncClient) syncFileContent(file FileIoProcessor, fileSize int64)
 	return nil
 }
 
-func (client *syncClient) sendHTTPRequest(method string, action string, interval Interval, data []byte) (*http.Response, error) {
+func (client *syncClient) sendHTTPRequest(method string, action string, interval Interval, data []byte, directIO bool) (*http.Response, error) {
```

Comment:
The new parameter `directIO` is required by the caller `client.openServer(directIO)` only. Asking other callers to set this option doesn't make sense. Hence, can we modify the input like the following?
```
sendHTTPRequest(method string, action string, querys map[string]string, data []byte)
```

Reply:
changed